x-pack/plugins/alerting/server/task_runner/rule_loader.ts

```diff
@@ -71,21 +72,22 @@ export function validateRule<Params extends RuleTypeParams>(
   if (!enabled) {
     throw new ErrorWithReason(
       RuleExecutionStatusErrorReasons.Disabled,
-      new Error(`Rule failed to execute because rule ran after it was disabled.`)
+      new Error(`Rule failed to execute because rule ran after it was disabled.`),
+      TaskErrorSource.FRAMEWORK
```

Comment:
Thinking about this error - I've seen it a lot in customer logs.

I think we intended that it would be a one-off - every now and again, somone would disable a rule while it was actually running - and it would result in this state.  

In reality, we see this with old-school zombie rules - the rules have disabled - but there's a task from before the task id === rule id - that keeps trying to run the rule.  It's been an interesting signal!  :-)

So, no other comment other than old me would have said this was USER error, but now I know it's more likely a FRAMEWORK error.